data/data/agent/files/usr/local/bin/start-cluster-installation.sh

```diff
@@ -40,6 +40,22 @@ num_known_hosts() {
         done
         echo "Hosts known and ready for cluster installation (${known_hosts}/${total_required_nodes})" 1>&2
     fi
+    if (( known_hosts > total_required_nodes )) && [[ "${TNA_CLUSTERS_SUPPORT}" == "true" ]]; then
```

Comment:
I would suggest adding a REQUIRED_ARBITER_NODES variable to https://github.com/openshift/installer/blob/main/data/data/agent/files/usr/local/share/start-cluster/start-cluster.env.template. That could simplify or eliminate this code block. The required master and workers are calculated here and passed to the template: https://github.com/openshift/installer/blob/55b35185c79710706be350d384fc35c2afb322c3/pkg/asset/agent/image/ignition.go#L180

Reply:
AgentClusterInstall is missing the arbiter role: https://github.com/openshift/assisted-service/blob/dafbf3354b6e2f587491ba3a96b352d63abebfbe/api/hiveextension/v1beta1/agentclusterinstall_types.go#L365

Reply:
Oh sure thing, way better approach, I got a PR open here https://github.com/openshift/assisted-service/pull/7765

Reply:
Added the change for now, until that PR is merged in